dart/dynamics/SimpleFrame.cpp

```diff
@@ -120,6 +132,13 @@ SimpleFrame& SimpleFrame::operator=(const SimpleFrame& _otherFrame)
 }
 
 //==============================================================================
+std::shared_ptr<SimpleFrame> SimpleFrame::createChildSimpleFrame(
```

Comment:
Are there any opinions about changing this function's name to ``spawnChildSimpleFrame``? Is anyone in favor of it or against it?